README.md

```diff
@@ -26,10 +26,11 @@ $ kubectl directpv info
 
 4. Discover and add drives for volume scheduling.
 ```sh
-# Discover drives
-$ kubectl directpv discover --output-file drives.yaml
+# Discover drives to check the available devices in the cluster to initialize
```

Comment:
This will discover all drives, but only mark those which are unformatted and non-root/loop as ready for DirectPV right?

Reply:
This will present only `Available` drives. The Available drives should comply with the following conditions.